server/index.js

```diff
@@ -8,7 +8,7 @@ const https = require("https");
 const fs = require("node:fs");
 
 const app = express();
-const port = 8000;
+const port = 8001;
```

Comment:
Just wondering why was changed to 8001?
This would require changes to the docker-compose.yml as well as .env file in `client`

Reply:
Oh sorry I forgot to change it back. I was testing it and I was getting Port 8000 was being used and I was impatient so I just changed it to 8001 while testing. I think it's okay to change it back to Port 8000. Sorry!